inst

```diff
@@ -245,19 +268,23 @@ nextcloud_ensure_extended_attributes () {
         --set valueRequired='0' \
         --set CLIName='nextcloudEnabled' \
         --set syntax='boolean' \
-        --set default="1" \
-        --set tabAdvanced='1' \
+        --set default="0" \
+        --set tabAdvanced='0' \
+        --set disableUDMWeb='1' \
+        --set options='nextcloudUser' \
         --set mayChange='1' \
         --set multivalue='0' \
         --set deleteObjectClass='0' \
         --set tabPosition='1' \
         --set overwritePosition='0' \
         --set doNotSearch='0' \
-        --set hook='None' || \
+        --set hook='' || \
     univention-directory-manager settings/extended_attribute modify "$@" \
         --dn "cn=nextcloudUserEnabled,cn=nextcloud,cn=custom attributes,cn=univention,$ldap_base" \
-        --set tabAdvanced='1' \
-        --set default="1" || die
+        --set tabAdvanced='0' \
+        --set disableUDMWeb='1' \
+        --set options='nextcloudUser' \
+        --set default="0" || die "Could not modify nextcloudUserEnabled extended attribute"
```

Comment:
See the comment above. Does this change default behavior for newly created users?